### PR TITLE
Fix bugs with bottom/right positioning.

### PR DESCRIPTION
**Pull request by jgerigmeyer**
- Fixes bugs with `bottom` or `right` positioning (needed to account for width/height of `offsetParent`)
- Uses utility fns from floating-ui instead of `computePosition` api

### CODE REVIEW

**Comment by jgerigmeyer** [comment]
[Fix bug with bottom/right positioning.](https://trello.com/c/mpbWQy5L/64-fix-bug-with-bottom-right-positioning)

**Review by mirisuzanne — COMMENTED** [review]
results lgtm

File: src/polyfill.ts [hunk 1]
```diff
@@ -56,65 +81,73 @@ export const getAxisProperty = (axis: 'x' | 'y' | null) => {
...
   }
 
+  const hasPercentage =
+    typeof percentage === 'number' && !Number.isNaN(percentage);
+
+  if (floatingPosition === 'bottom' || floatingPosition === 'right') {
+    offsetParent = getOffsetParent(floatingEl);
+    if (!isElement(offsetParent)) {
+      offsetParent = getDocumentElement(floatingEl);
+    }
```
Comment (jgerigmeyer) [inline src/polyfill.ts:135]: @sanajaved7 I figured this was a decent way to deal with the "initial containing block" situation? I don't really know what all this means, but it seems to work, in any case. 🤷 😅

Comment (sanajaved7) [reply src/polyfill.ts:135]: @jgerigmeyer ya I think this works! Looking at floating's `getDocumentElement` method more closely, the `window.document.documentElement` returns the `html` root element which is the initial containing block.

**Pull request merged**